api/src/main/java/org/cardanofoundation/rosetta/common/mapper/DataMapper.java

```diff
@@ -52,20 +50,23 @@
 public class DataMapper {
```

Comment:
Could you put it in separate methods Currency.builder() and Amount.builder() to maintain readability and avoid code duplication

Reply:
Do you mean moving the default currency to a separate method?
`Currency.builder()
        .decimals(Constants.ADA_DECIMALS)
        .symbol(Constants.ADA).build();`

Reply:
I will refactor it with the next PR, thanks for the comment.